master.js

```diff
@@ -361,27 +361,49 @@ $(function() {
     else {
       name = currentBrowserSelector;
     }
-    var results = table.find('tr:not([class*=test]):not(.optional-feature) td:not(.not-applicable)' + name);
-    var yesResults = results.filter('.yes').length;
-    var totalResults = results.length;
-    /*
-        Add annex b results
-    */
-    results = table.find('tr:not([class*=test]).optional-feature td:not(.not-applicable)' + name);
-    yesResults += results.filter('.yes').length;
-    totalResults += results.length;
-
-    var flaggedResults = yesResults;
-
-    table.find('tr.supertest td[data-tally]:not(.not-applicable)' + name).each(function() {
-      var weight = +$(this).parent().attr('significance') || 1;
-      var yes = (+$(this).attr('data-tally') || 0) * weight;
-      yesResults += yes;
-      flaggedResults += yes + (+$(this).attr('data-flagged-tally') || 0) * weight;
-      totalResults += weight;
+
+    var yesPoints = 0;
+    var flaggedPoints = 0;
+    var totalPoints = 0;
+
+    var rows = table.find('tbody tr:not([class=subtest]):not([class=category])');
+    rows.each(function() {
+      var row = $(this);
+      var points = Number(row.attr('significance') || 1);
+      var cell = row.children('td' + name);
+
+      if (row.hasClass('not-applicable')) {
```

Comment:
i think you can make this even faster by adding `:not(.not-applicable)` to the selector on line 369?